crates/sp-domains-fraud-proof/src/fraud_proof.rs

```diff
@@ -242,10 +246,10 @@ pub enum VerificationError<DomainHash> {
     InitializeBlockOrApplyExtrinsicDecode(codec::Error),
     /// Failed to decode the storage root produced by verifying `initialize_block` or `apply_extrinsic`.
     #[cfg_attr(
-        feature = "thiserror",
```

Comment:
I have the same issue with this missing white space when rust format removes them. I dont have a storng opinion either wway but having spaces is pleasing to look at though

Reply:
I agree, let me know if you find the way to make `rustfmt` not do this anymore, very annoying